packages/server-wallet/src/protocols/close-channel.ts

```diff
@@ -35,8 +43,23 @@ const signFinalState = (ps: ProtocolState): ProtocolResult | false =>
     isFinal: true,
   });
 
+const defundIntoLedger = (ps: ProtocolState): RequestLedgerDefunding | false =>
+  isLedgerFunded(ps.app) &&
+  !ps.ledgerDefundingRequested &&
+  !ps.ledgerDefundingSucceeded &&
+  !!ps.app.supported &&
+  ps.app.supported.isFinal &&
```

Comment:
This is incorrect: It is valid to transition to a `final` state, and thus we can have a supported `final` state.

Relevant: https://github.com/statechannels/statechannels/pull/2741#discussion_r511499088

Reply:
4cdd3154b